MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+    <ResourceDictionary>
+      <local:GroupCaptionConverter x:Key="SummaryConverter" />
+    </ResourceDictionary>
+  </ContentPage.Resources>
+
+  <sfgrid:SfDataGrid x:Name="dataGrid"
+                   ItemsSource="{Binding OrdersInfo}">
+
+<sfgrid:SfDataGrid.CaptionSummaryTemplate>
+   <DataTemplate>
+            <StackLayout Orientation="Horizontal">
+                <Label Text="{Binding Converter={StaticResource SummaryConverter}, ConverterParameter = {x:Reference dataGrid} }"
+        FontSize="Default"
+        VerticalTextAlignment="Center"
+        HorizontalTextAlignment="Start"
+        LineBreakMode="NoWrap"
+        HorizontalOptions="FillAndExpand"
+        VerticalOptions="FillAndExpand">
+                    <Label.Style>
+                        <Style TargetType="Label">
+                            <Setter Property="FontAttributes" Value="Bold, Italic" />
+                        </Style>
+                    </Label.Style>
+                </Label>
+            </StackLayout>
+        </DataTemplate>
+</sfgrid:SfDataGrid.CaptionSummaryTemplate>
+<sfgrid:SfDataGrid.CaptionSummaryRow>
+    <sfgrid:DataGridSummaryRow Name="CaptionSummary" ShowSummaryInRow="True" Title="Salary: {CaptionSummary}">
+        <sfgrid:DataGridSummaryRow.SummaryColumns>
+            <sfgrid:DataGridSummaryColumn Name="CaptionSummary"
+                                      Format="{}{Sum}"
+                                      MappingName="Salary"
+                                      SummaryType="DoubleAggregate" />
+        </sfgrid:DataGridSummaryRow.SummaryColumns>
+    </sfgrid:DataGridSummaryRow>
+</sfgrid:SfDataGrid.CaptionSummaryRow>
+</sfgrid:SfDataGrid>
+{% endhighlight%}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+
+// To write a converter, follow the code example:
+
+public class GroupCaptionConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var data = value != null ? value as Group : null;
+        if (data != null)
+        {
+            SfDataGrid dataGrid = (SfDataGrid)parameter;
+            var summaryText = SummaryCreator.GetSummaryDisplayTextForRow((value as Group).SummaryDetails, dataGrid.View);
+
+            return summaryText;
+        }
+        return null;
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return null;
+    }
+}
+
+
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with template loaded for entire caption summary row](Images\Caption-Summary\TemplateRow.png)
+
+N> The `DataTemplateSelector` can also be directly assigned to the `CaptionSummaryTemplate`.
+
+
+### Displaying template for a column
+
+The template for a Caption summary column can be set by using `DataGridSummaryColumn.Template` to customize it based on requirement. 
+
+Refer the below code example in which a label is loaded in the template of caption summary column.
```

Comment:
The template for a caption summary column can be customized by using the DataGridSummaryColumn.Template property. This allows you to define a custom template according to your requirements.

Here's an example code snippet that demonstrates how to load a label in the template of a caption summary column:

Reply:
Modified.